resources/dicts/relationship_events/normal_interactions/trust/decrease.json

```diff
@@ -1,207 +1,237 @@
+			"m_c is spreading a rumor about r_c."
+		],
+		"random_trait_constraint": [
+			"sneaky",
+			"childish",
+			"vengeful"
+		]
+	},
+	{
+		"id": "trust_kit_de_med1",
+		"interactions": [
+			"m_c constantly pulling pranks on r_c.",
+			"m_c always shifts the blame to r_c!",
+			"m_c won't stop crying whenever r_c starts winning in a game.",
+			"m_c grins while r_c is being scolded by an older Clanmate.",
+			"m_c is relentlessly pestering r_c."
+		],
+		"relationship_constraint": [
+			"dislike_40"
+		],
+		"main_status_constraint": [
+			"kitten"
+		],
+		"reaction_random_cat": {
+			"trust": "decrease",
+			"dislike": "increase"
+		}
+	},
+	{
+		"id": "trust_de_high1",
+		"intensity": "high",
+		"interactions": [
+			"m_c goes to tell a secret to r_c only for r_c to tell someone else's secret to {PRONOUN/m_c/object}.",
+			"r_c doesn't believe m_c when {PRONOUN/m_c/subject} {VERB/m_c/explain/explains} that a rumor about {PRONOUN/m_c/object} is false.",
+			"m_c learns that r_c has been using {PRONOUN/m_c/object} for something.",
+			"m_c doubts whether r_c even likes {PRONOUN/m_c/object} at all...",
+			"r_c uses a fear of m_c's against {PRONOUN/m_c/object}.",
+			"r_c accidentally hurts m_c during an argument between them.",
+			"m_c thinks r_c is trying to steal one of {PRONOUN/m_c/poss} friends!",
+			"m_c turned to r_c in a time of need, only for r_c to abandon {PRONOUN/m_c/object}."
+		]
+	},
+	{
+		"id": "trust_de_high2",
+		"intensity": "high",
+		"interactions": [
+			"m_c tries to scare r_c.",
+			"m_c accuses r_c of being in cahoots with a cat from another Clan.",
+			"m_c publicly accuses r_c of breaking the Warrior Code.",
+			"m_c is blindsighted by r_c completely betraying {PRONOUN/m_c/object}.",
+			"m_c threatens to blackmail r_c.",
+			"m_c and r_c refuse to trust one another, which nearly leads to one of them getting hurt.",
+			"r_c's prank on m_c almost turned deadly.",
+			"r_c has been pitting a Clanmate against m_c.",
+			"m_c claims to have had a prophetic dream about r_c betraying the Clan.",
+			"m_c and r_c tell everyone embarrassing secrets about the other.",
+			"r_c destroys something dear to m_c.",
+			"r_c blackmails m_c with one of {PRONOUN/m_c/poss} secrets."
+		],
+		"relationship_constraint": [
+			"dislike_40"
+		],
+		"reaction_random_cat": {
+			"trust": "decrease"
+		}
+	},
+	{
+		"id": "trust_for_medcat_de_high1",
+		"intensity": "high",
+		"interactions": [
+			"m_c can never seem explain {PRONOUN/r_c/poss} health in a way that makes r_c listen...",
+			"m_c feels like r_c is downplaying the severity of m_c's health concerns.",
+			"m_c accuses r_c of not taking care of cats properly.",
+			"m_c doesn't bother to ask r_c for help with a minor injury, worried that {PRONOUN/r_c/subject} might make it worse.",
+			"m_c complains that r_c doesn't seem to understand signs from StarClan very well.",
+			"m_c feels worse after r_c gave {PRONOUN/m_c/object} the wrong treatment for something minor."

```

Comment:
this should either be "the wrong treatment" _or_ "the treatment for something else", not both

Reply:
Rewrote to this for simplicity:

m_c thinks that r_c gave {PRONOUN/m_c/object} the wrong herbs.